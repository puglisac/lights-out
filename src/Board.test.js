import React from 'react';
import { render, fireEvent } from '@testing-library/react';
import Board from './board';

beforeEach(function() {
    jest
      .spyOn(Math, "random")
      .mockReturnValueOnce(0.25)
      .mockReturnValueOnce(1.75);
  });
  


test("renders without crashing", function() {
  render(<Board />);
});

test("matches snapshot", function() {
  const {asFragment} = render(<Board />);
  expect(asFragment()).toMatchSnapshot();
});


afterEach(function() {
    Math.random.mockRestore();
  });