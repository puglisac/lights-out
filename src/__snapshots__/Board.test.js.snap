// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`click works 1`] = `
<DocumentFragment>
  <table
    class="Board"
  >
    <tbody />
  </table>
</DocumentFragment>
`;

exports[`matches snapshot 1`] = `
<DocumentFragment>
  <table
    class="Board"
  >
    <tbody />
  </table>
</DocumentFragment>
`;
